### PR TITLE
Update Gentoo installation instructions for XDPH

### PR DESCRIPTION
**Pull request by foopsss**
* Changed the suggested overlay from useless-overlay to GURU.
* Bumped qt versions in the docs from 6.4.0 to 6.4.3 since 6.4.0 is not available anymore.

### CODE REVIEW

File: pages/Useful Utilities/Hyprland-desktop-portal.md [hunk 1]
```diff
@@ -38,29 +38,28 @@ yay -S xdg-desktop-portal-hyprland-git
...
 ### /etc/portage/profile/package.unmask
 ```plain
-=dev-qt/qtbase-6.4.0
-=dev-qt/qtwayland-6.4.0
-=dev-qt/qtdeclarative-6.4.0
-=dev-qt/qtshadertools-6.4.0
+=dev-qt/qtbase-6.4.3
+=dev-qt/qtwayland-6.4.3
+=dev-qt/qtdeclarative-6.4.3
+=dev-qt/qtshadertools-6.4.3
```
Comment (micielski) [inline pages/Useful Utilities/Hyprland-desktop-portal.md:44]: Matching on version is not needed in this case. So:
```dev-qt/qtbase
dev-qt/qtwayland
dev-qt/qtdeclarative
dev-qt/qtshadertools
```

File: pages/Useful Utilities/Hyprland-desktop-portal.md [hunk 2]
```diff
@@ -38,29 +38,28 @@ yay -S xdg-desktop-portal-hyprland-git
...
 ```plain
 gui-libs/xdg-desktop-portal-hyprland **
-=dev-qt/qtbase-6.4.0
-=dev-qt/qtwayland-6.4.0
-=dev-qt/qtdeclarative-6.4.0
-=dev-qt/qtshadertools-6.4.0
+=dev-qt/qtbase-6.4.3
+=dev-qt/qtwayland-6.4.3
+=dev-qt/qtdeclarative-6.4.3
+=dev-qt/qtshadertools-6.4.3
```
Comment (micielski) [inline pages/Useful Utilities/Hyprland-desktop-portal.md:62]: And same here

File: pages/Useful Utilities/Hyprland-desktop-portal.md (hunk 1 above)
Comment (foopsss) [reply pages/Useful Utilities/Hyprland-desktop-portal.md]: Done both. Is it good now?

File: pages/Useful Utilities/Hyprland-desktop-portal.md (hunk 2 above)
Comment (foopsss) [reply pages/Useful Utilities/Hyprland-desktop-portal.md]: Done.

**Comment by foopsss** [comment]
Well, this should be fine to merge now @vaxerski.

**Review by vaxerski — APPROVED** [review]
gangsta

**Pull request merged**